app/services/package.js

```diff
@@ -140,28 +146,46 @@ function convertResource(resource, dataPackage, dataPackageUrl) {
           field.title = originalField.title;
           field.slug = originalField.slug;
           field.type = originalField.osType;
+
+          // Populate additional properties
+          field.options = {};
+
           var measure = _.find(dataPackage.model.measures, function(item) {
             return item.source == field.name;
           });
           if (measure) {
             var excludeFields = ['resource', 'source', 'title'];
-            field.options = _.chain(measure)
-              .map(function(value, key) {
-                if (excludeFields.indexOf(key) == -1) {
-                  return [key, value];
-                }
-              })
-              .filter()
-              .fromPairs()
-              .value();
+            _.extend(field.options, _.chain(measure)
+            .map(function(value, key) {
+              if (excludeFields.indexOf(key) == -1) {
+                return [key, value];
+              }
+            })
+            .filter()
+            .fromPairs()
+            .value());
           }
+
+          var allowedFields = ['format', 'decimalChar', 'groupChar'];
+          _.each(dataPackage.model.dimensions, function(dimension) {
+            var attr = _.find(dimension.attributes, function(item) {
+              return item.source == field.name;
+            });
+            if (attr) {
+              _.extend(field.options, _.pick(attr, allowedFields));
+              // Field can belong only to one dimension, so once we found it -
```

Comment:
I added that comment to explain why I break iteration after processing attribute. Currently field can belong only to one dimension. If it will be changed in future - we may add some corrections to this code.